corpus/impulse/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from embedathon.models import PAYMENT_STATUS
+from accounts.models import User
+
+class ImpulseUser(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    from_nitk = models.BooleanField(default=True)
+    college_name = models.CharField(
+        max_length=200, default="National Institute of Technology Karnataka"
+    )
+    roll_no = models.CharField(max_length=8, blank=False, null=False)
+    ieee_member = models.BooleanField(default=False)
+    ieee_membership_no = models.BigIntegerField(blank=True, null=True)
+    team = models.ForeignKey(
+        "Team", on_delete=models.CASCADE, related_name="team", blank=True, null=True
+    )
+    is_member2 = models.BooleanField(default=False)
+    member2_name = models.CharField(max_length=200, blank=True, null=True)
+    member2_email = models.EmailField(blank=True, null=True)
+    member2_from_nitk = models.BooleanField(default=True)
+    member2_college_name = models.CharField(
+        max_length=200, default="National Institute of Technology Karnataka"
+    )
+    member2_roll_no = models.CharField(max_length=8, blank=True, null=True)
+    member2_phone = models.CharField(max_length=13, blank=True, null=True)
+    member2_ieee_member = models.BooleanField(default=False)
+    member2_ieee_membership_no = models.BigIntegerField(blank=True, null=True)
```

Comment:
What is this `member2` logic? Did not understand this. If your requirement is that every team should have two people or something, have that in the team model (as `leader` and `member`). This way of storing data is not making sense.

Reply:
Okay so there can be max of 2 members per team, I thought the invite sending can be overkill just for a 2 team event. 
Yes the member2 logic is weird, my bad. I have put it as member_details (each field listed out) under Team model, is that okay ? or is it better following the invite method ?